qiita_pet/templates/study_base.html

```diff
@@ -0,0 +1,72 @@
+{% extends sitebase.html %}
+{% block head %}
+<script type="text/javascript">
+  function fill_base(sid) {
+  $.get( "/study/description/baseinfo/", { study_id: sid} )
+    .done(function( data ) {
+      $("#study-main").html(data);
+      $("#study-prep-graph").hide();
+      $("#study-file-breadcrumbs").hide();
+    });
+  }
+
+  $(document).ready(function() {
+    fill_base({{study_info['study_id']}});
+  });
+</script>
+{% end %}
+{% block content %}
+<div class="row">
+  <div class="col-md-3">
+  <h3>Study Information</h3>
+    <a href="#">Sample Template</a><br />
```

Comment:
I agree. I've added an if statement to show certain things only if study is yours or shared with you.